commands/proposal_submission.go

```diff
@@ -16,6 +16,7 @@ import (
 	vegapb "code.vegaprotocol.io/vega/protos/vega"
 	commandspb "code.vegaprotocol.io/vega/protos/vega/commands/v1"
 	datapb "code.vegaprotocol.io/vega/protos/vega/data/v1"
+	"github.com/shopspring/decimal"
```

Comment:
Not really a fan of this `decimal` package leaking here. It's quite an expensive type to use, so if we want to swap it out for something better/faster at some point, it makes sense to stick to the `num.Decimal` type as much as possible

Reply:
ah my bad, I thought I had removed them all. leme fix.